docker-compose.test.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.4"
```

Comment:
These dependencies are already met by `docker-compose.yml`. Can you expand on why we need a dedicated test one?

Reply:
`docker-compose.yml` bring dependencies + website up for testing and only open 8000 to show the website. `docker-compose.test.yml` goal is to only spin up docker + redis so you have a local instances the tests can connect to.

There may be a better way to do that

Reply:
The way I would probably run it is:

```
docker-compose run web python -m pytest tests/
```

It may be worth throwing in a makefile to simplify commands (see #374)